Classes/Export/Event/AlternateExportQueryEvent.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Calien\Xlsexport\Export\Event;
+
+use Psr\EventDispatcher\StoppableEventInterface;
+
+final class AlternateExportQueryEvent implements StoppableEventInterface
+{
+    /**
+     * @var string
+     */
```

Comment:
superfluous DocBlock, please remove

Reply:
Removed in 15e899cad26a2df606b5af0bff2cf8764146a5ab